docs/api/utility.rst

```diff
@@ -0,0 +1,325 @@
+.. # Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. # other BLT Project Developers. See the top-level COPYRIGHT file for details
+.. # 
+.. # SPDX-License-Identifier: (BSD-3-Clause)
+
+Utility Macros
+==============
+
+
+General Utility Macros
+----------------------
+
+
+blt_list_append
+~~~~~~~~~~~~~~~
+
+.. code-block:: cmake
+
+    blt_list_append( TO <list> ELEMENTS [ <element>...] IF <bool> )
+
+Appends elements to a list if the specified bool evaluates to true.
+
+This macro is essentially a wrapper around CMake's `list(APPEND ...)`
```

Comment:
I think these need to be double quoted to display properly in sphinx.
I.e.
```
``list(APPEND ...)``
```
instead of
```
`list(APPEND ...)`
```

Reply:
fixed.